packages/web-config-server/src/apiV1/dataBuilders/generic/table/tableOfDataValues/tableOfCalculatedValues.js

```diff
@@ -24,8 +24,9 @@ class TableOfCalculatedValues extends TableOfDataValuesBuilder {
     );
   }
 
-  buildValuesByCell() {
-    return getCalculatedValuesByCell(flatten(this.tableConfig.cells), this.results);
+  async buildValuesByCell() {
```

Comment:
I don't really like the idea of making this async, but is there a cleaner way of doing it?